its/plugin/projects/phpunit/reports/phpunit.coverage-empty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1234543251" phpunit="3.3.1">
+  <project name="AllTests" timestamp="1234543251">
+  </project>
+</coverage>
```

Comment:
I would cal this file something like `no-record` because we use `empty` for real empty files.